pbm/log/log.go

```diff
@@ -23,19 +24,20 @@ type Logger struct {
 }
 
 type LogEntry struct {
-	TS      int64 `bson:"ts" json:"ts"`
-	Tns     int   `bson:"ns" json:"ns"`
-	TZone   int   `bson:"tz" json:"tz"`
+	Time    string `bson:"-" json:"t"`
+	TS      int64  `bson:"ts" json:"-"`
+	Tns     int    `bson:"ns" json:"-"`
+	TZone   int    `bson:"tz" json:"-"`
 	LogKeys `bson:",inline" json:",inline"`
 	Msg     string `bson:"msg" json:"msg"`
 }
 
 type LogKeys struct {
-	RS      string    `bson:"rs" json:"rs"`
-	Node    string    `bson:"node" json:"node"`
-	Type    EntryType `bson:"type" json:"type"`
-	Event   string    `bson:"event" json:"event"`
-	ObjName string    `bson:"obj" json:"obj"`
+	Severity Severity `bson:"s" json:"s"`
+	RS       string   `bson:"rs" json:"rs"`
+	Node     string   `bson:"node" json:"node"`
+	Event    string   `bson:"e" json:"e"`
+	ObjName  string   `bson:"obj" json:"obj"`
```

Comment:
```suggestion
	ObjName  string   `bson:"obj" json:"eobj"`
```
"obj" is too broad.

Alternatively what is currently "e" could event type ("etype"), and this could be "e" (or "event", or "eobj").

Reply:
https://github.com/percona/percona-backup-mongodb/pull/441/commits/c31c3282fad2f64e8e0c0a480e410a97edca9992